src/config.ts

```
@@ -1,47 +1,20 @@
+  openaiApiKey: process.env.OPENAI_API_KEY || '',
+  // The WebSocket endpoint for realtime transcription, per the latest OpenAI docs.
+  openaiEndpoint: process.env.OPENAI_ENDPOINT || 'wss://api.openai.com/v1/realtime/transcriptions',
 
-// API keys
-export const apiKeys = {
-  meetingBaas: process.env.MEETING_BAAS_API_KEY || "",
-  gladia: process.env.GLADIA_API_KEY || "",
-};
+  // Proxy and MeetingBaas configuration
+  proxyHost: process.env.PROXY_HOST || '0.0.0.0',
+  proxyPort: parseInt(process.env.PROXY_PORT || '3000', 10),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)